bubble_sort/Cpp/bubble_sort.cpp

```diff
@@ -1,4 +1,5 @@
-#include <stdio.h>
+#include <iostream>
+using namespace std;
 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpv36ui5mt/bubble_sort/Cpp/bubble_sort.cpp
+++ b/tmp/tmpv36ui5mt/bubble_sort/Cpp/bubble_sort.cpp
@@ -18,7 +18,7 @@
 				swapped = true;
 			}
 		}
-		
+
 		if (swapped == false)
 			break;
 	}
```